### PR TITLE
Add instructions for disabling splash screen effects

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @LarkAscending

File: assets-and-images.md [hunk 1]
```diff
@@ -248,10 +248,14 @@ Flutter also uses native platform mechanisms to draw transitional launch screens
 
 To add a "splash screen" to your Flutter application, navigate to `.../android/app/src/main`. In `res/drawable/launch_background.xml`, You can use this [layer list drawable](https://developer.android.com/guide/topics/resources/drawable-resource.html#LayerList) XML to customize the look of your launch screen. The existing template provides an example for adding a image to to the middle of a white splash screen in commented code. You can uncomment it or use other [drawables](https://developer.android.com/guide/topics/resources/drawable-resource.html) to achieve the intended effect.
```
Comment (LarkAscending) [inline assets-and-images.md]: for adding *an* image to ~~to~~

File: assets-and-images.md [hunk 2]
```diff
@@ -248,10 +248,14 @@ Flutter also uses native platform mechanisms to draw transitional launch screens
...
 
+You can disable the effect of showing the splash screen until Flutter renders the first frame by setting the default first activity's meta-data `io.flutter.app.android.SplashScreenUntilFirstFrame` value to false in the `AndroidManifest.xml` or by not setting a `android:windowBackground` altogether on the activity via the default `@style/LaunchTheme` theme.
```
Comment (LarkAscending) [inline assets-and-images.md]: You can delay showing the splash screen until right after Flutter renders the first frame of your app. <does this help with apps that are already fast to launch? does it help remove the default blank screen?>. To delay the splash screen, modify the default first activity entry in AndroidManifest.xml in one of the following ways:
* Set the value of `io.flutter.app.android.SplashScreenUntilFirstFrame` to `false`
* If the `android:theme` attribute exists, do not set `android:windowBackground` for the default `@style/LaunchTheme` theme

File: assets-and-images.md [hunk 3]
```diff
@@ -248,10 +248,14 @@ Flutter also uses native platform mechanisms to draw transitional launch screens
...
+
 #### iOS
 
-To add an image to the center of your "splash screen", navigate to `.../ios/Runner`. In, `Assets.xcassets/LaunchImage.imageset`, drop in images named `LaunchImage.png`, `LaunchImage@2x.png`, `LaunchImage@3x.png`. If you use different filenames, you'll also have to update the `Contents.json` file in the same directory.
+To add an image to the center of your "splash screen", navigate to `.../ios/Runner`. In, `Assets.xcassets/LaunchImage.imageset`, replace the placeholder images in the directory. If you use different filenames, you'll also have to update the `Contents.json` file in the same directory.
```
Comment (LarkAscending) [inline assets-and-images.md]: Good changes!

* Remove quotes around `splash screen`
* Remove comma after `In`
* think about changing `different` to `non-default` or `custom`

File: assets-and-images.md [hunk 4]
```diff
@@ -248,10 +248,14 @@ Flutter also uses native platform mechanisms to draw transitional launch screens
...
 
 You can also fully customize your launch screen storyboard in Xcode by opening `.../ios/Runner.xcworkspace`. Navigate to `Runner/Runner` in the Project Navigator and drop in images by opening `Assets.xcassets` or do any customization using the Interface Builder in `LaunchScreen.storyboard`.
 
 ![Adding launch icons in Xcode](/images/assets-and-images/ios-launchscreen-xcode.png)
+
```
Comment (LarkAscending) [inline assets-and-images.md]: * change `drop in` to `add or replace`
* Add period after `.xcassets`
* `For more complex changes, you can also use the Interface Builder in LaunchScreen.storyboard`

* change `un-setting` to `removing the value of` or `setting the Launch Screen File field to null`
*  Use path notation for UI coordinates, like `Runner\Runner>General>App Icons and Launch Images` (no need for words like `settings` or `section`

**Comment by mit-mit** [comment]
@xster still working on this?

**Comment by Hixie** [comment]
@xster I'm going to close this PR to get it off the review queue since it's not ready to land. Please don't hesitate to reopen it when you're ready for final review and landing. Thanks!

**Pull request closed without merging**